index.bs

```diff
@@ -1517,6 +1517,7 @@ Return the index location of the minimum or maxmium values of all the input valu
 <script type=idl>
 dictionary MLArgMinMaxOptions {
   boolean keepDimensions = false;
+  MLOperandDataType outputDataType = "int32";
```

Comment:
`uint32`? That way:
- it's consistent with WebNN in general, where everything related to axes and dimensions is uint32 (the one notable exception being gather which allows negative indices).
- it's possible to represent the full range of a single 1D tensor (otherwise we lose half the addressable space).
- when going from int64 down to uint32, we're less likely to chop of the upper bit.

Of course, passing `int32` would still be valid for CoreML usage.

Reply:
Let's see...
- DML accepts both uint32 and int32: https://learn.microsoft.com/en-us/windows/win32/api/directml/ns-directml-dml_argmin_operator_desc
- TF evidently only accepts output_type int32 (not uint32) https://www.tensorflow.org/api_docs/python/tf/math/argmin.
- CoreML only returns int32 https://apple.github.io/coremltools/source/coremltools.converters.mil.mil.ops.defs.html#coremltools.converters.mil.mil.ops.defs.iOS15.reduction.reduce_argmin.

Well shoot, I guess it should stay `int32` due to backend limitations, meaning we'll only be able to access half of the addressable space.

Reply:
thanks @fdwr  I see your comments in the [CL](https://chromium-review.googlesource.com/c/chromium/src/+/5692538/comment/062884c1_054aad70/)  too. I will reply it here. 

> should the default parameter of an API favor that of the caller viewpoint (frontends) or the implementations (backends)?

If `int64` is emulatable on CoreML then we could possibly favor values that make it easier for callers. The problem is `int64` is not emulatable on CoreML so it will just fail, breaking browser compatibility as [Reily mentioned](https://chromium-review.googlesource.com/c/chromium/src/+/5692538/comment/062884c1_054aad70/). So I think it's better to default to a value that just works on all implementations?